tests/fullydistributed_grids/copy_distributed_tria_01.cc

```diff
@@ -30,7 +30,7 @@
 #include <deal.II/grid/tria.h>
 #include <deal.II/grid/tria_description.h>
 
-#include "./tests.h"
+#include "../grid/tests.h"
```

Comment:
?

Reply:
The `tests.h` file has just been moved from `fullydistributed_grids` to `grids`, without modify its content.

Is it fine to reference it also from other folders other than `grids`?